### PR TITLE
Adds documentation for Saved Actions

### PR DESCRIPTION
**Pull request by asimshah25**
Hold on merge until the feature is released.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by aliciascott — CHANGES_REQUESTED** [review]
Made some small suggestions, also should this page be linked from service_management/workflows/actions_catalog/ page? If so please add this new page to a link from there, or add it to the left - hand navigation below "Generic Actions"
Instructions for updating Left - hand nav: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/2125332970/New+Documentation+Page
If you need help with this just let me know.
Re-request review once you are ready to merge/approve!

**Comment by asimshah25** [comment]
> Made some small suggestions, also should this page be linked from service_management/workflows/actions_catalog/ page? If so please add this new page to a link from there, or add it to the left - hand navigation below "Generic Actions" Instructions for updating Left - hand nav: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/2125332970/New+Documentation+Page If you need help with this just let me know. Re-request review once you are ready to merge/approve!

Thank you so much for the quick and thorough review! I have made the changes as suggested and also a couple of tweaks of my own. Please let me know if I messed up something there.
I have also added the page to the left-hand navigation under Generic Actions. I hope I did it correctly though 🤦

**Review by sarjyusuf — CHANGES_REQUESTED** [review]

File: content/en/service_management/workflows/actions_catalog/saved_actions.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+title: Store and Reuse Actions
+kind: documentation
+disable_toc: false
+disable_sidebar: false
+further_reading:
+- link: "/integrations/"
+  tag: "Documentation"
+  text: "Learn about integrations"
+---
+
```
Comment (rtrieu) [inline content/en/service_management/workflows/actions_catalog/saved_actions.md:11]:
```suggestion
## Overview

```

File: content/en/service_management/workflows/actions_catalog/saved_actions.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+title: Store and Reuse Actions
```
Comment (rtrieu) [inline content/en/service_management/workflows/actions_catalog/saved_actions.md:2]: For some reason the title is appearing as an H2 instead of a Title.

**Review by rtrieu — CHANGES_REQUESTED** [review]
A few minor suggestions. I also couldn't figure out why the title is appearing as an H2 instead of a Title on the page.

File: content/en/service_management/workflows/actions_catalog/saved_actions.md (hunk 2 above)
Comment (hestonhoffman) [reply content/en/service_management/workflows/actions_catalog/saved_actions.md:2]: This was happening to me in preview and I thought I'd fixed it. I'll take another look

Comment (hestonhoffman) [reply content/en/service_management/workflows/actions_catalog/saved_actions.md]: Fixed it. The `cascade` on the index page was causing the issue.

**Review by hestonhoffman — APPROVED** [review]

**Review by rtrieu — APPROVED** [review]
Looks great!

**Pull request merged**